gflownet/envs/base.py

```diff
@@ -528,6 +553,20 @@ def get_logprobs(
         logprobs = self.logsoftmax(logits)[ns_range, action_indices]
         return logprobs
 
+    def get_jacobian_diag(
```

Comment:
It is not needed actually. I think I added it (back in spring) thinking of a future `ContinuousEnv` (or the like) to explicitly indicate that it may be necessary to compute the Jacobian.

Reply:
So perhaps just remove it for now?

Reply:
Ok I'll remove it, at the risk of forgetting to mention it later.

Reply:
https://github.com/alexhernandezgarcia/gflownet/pull/204/commits/9cd47a6e186f79002d66f00463e231c26b89c65a